mango-viz/bdgenomics/mango/io/test/test_bed.py

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to Big Data Genomics (BDG) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The BDG licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+from bdgenomics.mango.io import *
+
+class BedFileTest(unittest.TestCase):
+
+    dataframe = io.read_bed("../../../examples/data/chr17.582500-594500.bed")
+
+    def test_columns():
+        dataframe_columns = list(dataframe.columns)
+        for name in ("chrom", "chromStart", "chromEnd"):
+            assert(name in dataframe_columns)
+
+    def test_correct_format():
+        chromosomes = list(dataframe["chrom"])
+        chromStart = list(dataframe["chromStart"])
+        chromEnd = list(dataframe["chromEnd"])
+        for item in chromsomes:
+            assert(type(item) == str)
+        for item in chromStart:
+            assert(type(item) == int)
+        for item in chromEnd:
+            assert(type(item) == int)
+        
```

Comment:
Some other unit test ideas include:

def test_from_dataframe(): make sure  a user can take a dataframe and convert it to a bed dataframe

You can also test:
- that the number or rows are correct
- that the json strings are correct

Reply:
Currently, the read_bed functionality requires the input to be a path destination to a .bed file. User's cannot take an existing dataframe and convert it to a bed dataframe. Is that functionality that we want added ?

Reply:
I believe this functionality already exists by using `def _parse(cls, df)` in mango-viz/bdgenomics/mango/io/bedfile.py. This should be testable.